packages/table/src/cell/editableCell.tsx

```diff
@@ -8,14 +8,19 @@
 import * as classNames from "classnames";
 import * as React from "react";
 
-import { EditableText, Utils as CoreUtils } from "@blueprintjs/core";
+import { EditableText, Hotkey, Hotkeys, HotkeysTarget, Utils as CoreUtils } from "@blueprintjs/core";
 
 import * as Classes from "../common/classes";
 import { Draggable } from "../interactions/draggable";
 import { Cell, ICellProps } from "./cell";
 
 export interface IEditableCellProps extends ICellProps {
     /**
+     * Whether or not the given cell is the current active/focused cell.
```

Comment:
Do we need the `or not`?

https://english.stackexchange.com/questions/3382/whether-or-not-vs-whether